plugins/times-of-day/frontend/public/templates/widget.html

```diff
@@ -1,51 +0,0 @@
-<div class="grid-stack-item-content cly-widget">
```

Comment:
this file will also be required in order for the times of day widget to work on custom dashboards.